src/libos/src/fs/sefs/sgx_storage.rs

```diff
@@ -10,6 +12,18 @@ use std::sgxfs::{remove, OpenOptions, SgxFile};
 use std::sync::{Arc, SgxMutex as Mutex};
 use std::untrusted::fs;
 
+/// Helper macro to convert the std::io::Result to rcore_fs::dev::DevResult.
+///
+/// Implementing a From<T> trait is disallowed here, because both types are not defined in this crate.
+/// This macro uses a closure to convert the std::io::Result to error::Result at first,
+/// then converts the error::Result to rcore_fs::dev::DevResult.
+macro_rules! convert_to_devresult {
```

Comment:
Since this macro is not limited to converting to `DevResult`, I think it is fair to call it `convert_result`.